docs/validators/get-started/initialize-node.md

```diff
@@ -31,10 +31,10 @@ Update the genesis file.
 curl http://mainnet.regen.network:26657/genesis | jq .result.genesis > ~/.regen/config/genesis.json
 ```
 
-*For Redwood Testnet:*
+*For Regen Testnet:*
 
 ```bash
-curl http://redwood.regen.network:26657/genesis | jq .result.genesis > ~/.regen/config/genesis.json
+curl https://rpc-regen-upgrade.vitwit.com/genesis | jq .result.genesis > ~/.regen/config/genesis.json
 ```
 
 ## Update Peers
```

Comment:
@anilcse @kaustubhkapatral could you help with that section as there are no peers currently for the regen-upgrade testnet node
we have also been discussing about running a full node in house for the new testnet (although not prioritized yet cc/ @S4mmyb) so we could update our docs to use it instead and set your node as peer if that makes sense

Reply:
You can use this as a peer for now. `fc6cf74d3de04ab0a3836f5adf50968990f2c195@155.138.196.253:26656`